docs/sdkx-react-native/sdk-configuration.mdx

```diff
@@ -0,0 +1,306 @@
+---
+sidebar_position: 30
+title: SDK Configuration
+description: "Helpshift provides several config options which can be used to customize the behaviour of the SDK."
+---
+
+import {
+  Admonition,
+  CodeBlock,
+  Tabs,
+  TabItem,
+  LatestSdkVersion,
+  Centered,
+  Image,
+  Intro,
+  SideBySide,
+  DownloadButton,
+  Steps,
+  Step,
+} from "@site/src/components/forDocs";
+
+# SDK Configuration
+
+<Intro>
+
+Helpshift provides several config options which can be used to customize the behaviour of the SDK.
+
+</Intro>
+
+## Install call configurations
+
+### Enable Logging {#enableLogging}
+
+Upon setting `enableLogging` to `true`, Helpshift SDK logs will be generated in the Android logcat/ Xcode console. Turning on logging can help developers resolve common integration issues.
+
+<div className="compact ">
+
+|         |                                     |
+| ------- | ----------------------------------- |
+| Key     | `enableLogging` |
+| values  | true/false                          |
+| default | true                                |
+
+</div>
+
+```
+let installConfig = {
+  'enableLogging': true,
+}
+let appId = Platform.select({
+  android: <Android_Platform_Id>,
+  ios: <iOS_Platform_Id>,
+});
+
+if (appId != null) {
+  install(appId, <domain>, installConfig);
+} else {
+  //Display your error here.
+}
+```
+### Enable In-App Notifications {#enable-in-app-notifications}
+
+If you do not want the in-app notification support provided by the Helpshift SDK, you can set the flag to `false`.
+
+For Android, setting this flag as `false` will stop showing notifications in the notification tray of the device but it will fetch messages in the background.
+
+For iOS, setting this flag as `false` will just disable the in-app notifications, push notifications will still be shown in the device’s notification center.
+
+The default value of this flag is `true` i.e., the in-app notification will be enabled.
+
+<div className="compact ">
+
+|         |                           |
+| ------- | --------------------------|
+| Key     | `enableInAppNotification` |
+| values  | true/false                |
+| default | true                      |
+
+</div>
+
+```
+let installConfig = {
+  'enableInAppNotification': true,
+}
+let appId = Platform.select({
+  android: <Android_Platform_Id>,
+  ios: <iOS_Platform_Id>,
+});
+
+if (appId != null) {
+  install(appId, <domain>, installConfig);
+} else {
+  //Display your error here.
+}
+```
+
+## Full privacy {#fullprivacy}
+
+In the `config` dictionary of the `showConversation` API at the time of calling this API, setting the `fullPrivacy` option to `true` ensures COPPA compliance by:
+
+<div className="compact ">
```

Comment:
The table is out of place. It breaks into the flow of the previous line and the lines following it.

Reply:
Updated and place at end.